pkg/manager/manager.go

```diff
@@ -149,6 +149,8 @@ type Options struct {
 	// It is used to set webhook.Server.Host.
 	Host string
 
+	// CertDir is the directory that contains the server key and certificate.
```

Comment:
We should make the GoDoc more clear that the CertDir is optional. If not set, default will be used.
Ref: https://github.com/kubernetes-sigs/controller-runtime/blob/7b08fd6a95a05eec0bc9c62bb5e77c11bac605c4/pkg/webhook/server.go#L87-L89

Reply:
done